src/js/edu-benefits/5490/config/form.js

```diff
@@ -459,11 +446,7 @@ const formConfig = {
                     properties: {
                       name: highSchool.properties.name,
                       city: highSchool.properties.city,
-                      state: highSchool.properties.state,
-                      dateRange: highSchool.properties.dateRange,
```

Comment:
OK, sounds like taking out `dateRange` is correct since it's never needed, but I think `hours`, `hoursType`, and `degreeReceived` are still needed when the choice is `graduated`.